conf/nomad/minio.hcl

```diff
@@ -36,6 +43,12 @@ job "${service_name}" {
     task "server" {
       driver = "docker"
 
+      volume_mount {
+        volume      = "persistence"
+        destination = "/local/data"
```

Comment:
I think it should be input variable

Reply:
I have added a new input variable called data_dir to address this for now. However, as per the conversation we had over skype, it would be good to revisit the minio amd mc nomad job files and refactor them to make them more user-friendly. Added this issue to gather thoughts #37